circle.yml

```diff
@@ -22,6 +23,11 @@ test:
   # Grep error on the documentation
   override:
     - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
+deployment:
+  push:
+    branch: master
+    commands:
+      - bash continuous_integration/push_doc.sh
```

Comment:
Will this mean that the deployment script will only be triggered after a push / merge to the master branch and never on a non-merged PR, even if we remove the `general / branches / only / master` restriction below?

Maybe we need a safety net in the `continuous_integration/push_doc.sh` script itself to ensure that?